src/cascadia/TerminalApp/ActionArgs.h

```diff
@@ -132,7 +127,7 @@ namespace winrt::TerminalApp::implementation
             auto otherAsUs = other.try_as<NewTabArgs>();
             if (otherAsUs)
             {
-                return otherAsUs->_TerminalArgs == _TerminalArgs;
+                return otherAsUs->_TerminalArgs.Equals(_TerminalArgs);

```

Comment:
I think we actually need a less strict check than this, right? like: if I bind a profile guid, name OR index to newTab, should it show up in the list?